pkg/providers/instancetype/suite_test.go

```diff
@@ -514,7 +514,7 @@ var _ = Describe("InstanceType Provider", func() {
 			Expect(kubeletFlags).To(ContainSubstring("--eviction-hard=memory.available<750Mi"))
 			Expect(kubeletFlags).To(ContainSubstring("--eviction-soft=memory.available<1Gi"))
 			Expect(kubeletFlags).To(ContainSubstring("--eviction-soft-grace-period=memory.available=10s"))
-			Expect(kubeletFlags).To(ContainSubstring("--max-pods=100")) // kubenet
+			Expect(kubeletFlags).To(ContainSubstring("--max-pods=250")) // networkPlugin=azure
```

Comment:
Can we also add a case to cover kubenet and validate we are still seeing 100 as max pods for kubenet?